pkg/operator/ceph/file/filesystem.go

```diff
@@ -140,6 +140,26 @@ func newFS(name, namespace string) *Filesystem {
 	}
 }
 
+// SetPoolSize function sets the sizes for MetadataPool and dataPool
+func SetPoolSize(f *Filesystem, context *clusterd.Context, spec cephv1.FilesystemSpec) error {
+	// generating the metadata pool's name
+	metadataPoolName := generateMetaDataPoolName(f)
+	err := client.CreatePoolWithProfile(context, f.Namespace, metadataPoolName, spec.MetadataPool, appName)
+	if err != nil {
+		return errors.Wrapf(err, "failed to update metadata pool %q", metadataPoolName)
+	}
+	// generating the data pool's name
+	dataPoolsName := generateDataPoolNames(f, spec)
```

Comment:
nit: let's name it `dataPoolNames` so it matches the method name

Reply:
Done